handbook/people-ops/from-graphbook/holidays.md

```diff
@@ -1,17 +1,15 @@
-# Holiday schedule
+# Holidays
 
-The Company’s office is closed on the following dates: 
+We have a team that is based throughout many parts of the world and we respect everyone's local holidays. Therefore, there is no need for teammates to provide a specific enumeration of holidays by country, as holidays are explicitly covered as part of our unlimited PTO policy. 
 
-* Observed January 1st - New Years Day
+Teammates should add their respective holidays to the "Not working" calendar in order to communicate to the team that they will be unavailable for those particular dates. 
```

Comment:
```suggestion
Teammates should add their respective holidays to the "Not working" calendar as far in advance as possible in order to communicate to the team that they will be unavailable for those particular dates. 
```

I know we talked about this, but I can't recall what we settled on. Was it just leaving it out entirely?